src/lib/blueprint.ts

```diff
@@ -403,40 +405,47 @@ const createResponse = (responses: OpenapiOperation['responses']): Response => {
   }
 }
 
+const PropertySchema = z
```

Comment:
I think we should only use zod for the openapi spec parsing and leave blueprint as a normal TypeScript def. Zod is a parse lib, and we only need to parse and normalize the openapi spec. We can set the defaults for missing stuff there.